pkg/entities/types.go

```diff
@@ -7996,6 +7998,12 @@ type DashboardVariable struct {
 	Type DashboardVariableType `json:"type,omitempty"`
 }
 
+// DashboardVariableOptionsInput - Options applied to the variable
+type DashboardVariableOptionsInput struct {
+	// Only applies to variables of type NRQL. With this turned on, the time range for the NRQL query will override the time picker on dashboards and other pages. Turn this off to use the time picker as normal.
+	IgnoreTimeRange bool `json:"ignoreTimeRange,omitempty"`
+}
+
```

Comment:
```suggestion
// DashboardVariableOptions - Options applied to the variable
type DashboardVariableOptions struct {
	// Only applies to variables of type NRQL. With this turned on, the time range for the NRQL query will override the time picker on dashboards and other pages. Turn this off to use the time picker as normal.
	IgnoreTimeRange bool `json:"ignoreTimeRange,omitempty"`
}
```

Reply:
Upon seeing all the above changes I realised there is a gap between tutone generation and manual modification. I will check if these manual changes need to be accommodated